backend/src/data_entry/validation.rs

```diff
@@ -436,6 +459,62 @@ impl Validate for PollingStationResults {
     }
 }
 
+impl Validate for ExtraInvestigation {
+    fn validate(
+        &self,
+        _election: &ElectionWithPoliticalGroups,
+        _polling_station: &PollingStation,
+        validation_results: &mut ValidationResults,
+        path: &FieldPath,
+    ) -> Result<(), DataError> {
+        if self.extra_investigation_other_reason.is_answered()
+            != self.ballots_recounted_extra_investigation.is_answered()
```

Comment:
```suggestion
        if self.extra_investigation_other_reason.is_answered()
            && !self.ballots_recounted_extra_investigation.is_answered()
```
`&& !` over `!=` is a bit clearer I think (or will it be incorrect then?)

Reply:
Fair point :+1: